### PR TITLE
fn: test and release scripts now run some jobs in background

### PR DESCRIPTION
**Pull request by skinowski**

### CODE REVIEW

**Comment by skinowski** [comment]
Not sure how I feel about this change myself due to the mess it introduces into shell scripts, but it does speed things a bit.

**Comment by treeder** [comment]
ehhh, not sure how I feel about it either. Maybe we can wait until CI gets way out of control before we start adding this?  Also, Circle supports parallel steps that might make it easier to grok when trying to figure out where something went wrong: https://circleci.com/docs/2.0/parallelism-faster-jobs/

File: test.sh [hunk 1]
```diff
@@ -23,19 +25,33 @@ function host {
...
-docker pull fnproject/hello
+( docker pull fnproject/sleeper && docker pull fnproject/error && docker pull fnproject/hello) &
+PID2=$!
+
+# pull db images
+( docker pull postgres && docker pull mysql && docker pull minio/minio) &
+PID3=$!
+
+# pull swagger image
+(docker pull quay.io/goswagger/swagger ) &
```
Comment (denismakogon) [inline test.sh]: see my comments at related issue, this image huge, why not just download binary for releases?

**Comment by rdallman** [comment]
seems like we got this down to 7 minutes for non-master builds in #624 and #629 will probably improve a little more once that's in, let's punt on this?

**Pull request closed without merging**